test/tools/jsontests/BlockChainTestsBoost.cpp

```diff
@@ -173,4 +173,36 @@ BOOST_AUTO_TEST_CASE(userDefinedFile)
 	dev::test::userDefinedTest(dev::test::doBlockchainTests);
 }
 BOOST_AUTO_TEST_SUITE_END()
+
+//BlockChainTests
+BOOST_AUTO_TEST_SUITE(BlockChainTestsRandom)
+
+BOOST_AUTO_TEST_CASE(bcRandom)
```

Comment:
BlockChainTestsBoost.cpp   
file contains boost test cases for blockchain tests